app/components/base_field/base_field.html.erb

```diff
@@ -11,15 +11,15 @@
     <% if help %>
       <p class="BaseField-helpText"><%= help %></p>
     <% end %>
-
-    <% if errors.any? %>
-      <% errors.each do |error| %>
-        <strong class="BaseField-errorText">
-          <%= error %>
-        </strong>
-      <% end %>
-    <% end %>
   </div>
 
   <%= content %>
+
+  <% if errors.any? %>
+    <% errors.each do |error| %>
+      <strong class="BaseField-errorText">
+        <%= error %>
+      </strong>
+    <% end %>
+  <% end %>
```

Comment:
I made this change because the `hide_label` attribute was hiding any errors that occur and I don't think that was its intention. At least semantically that is not what it says it does. Also, I am more familiar with the errors on the bottom of the input vs. the top